_posts/2014-02-20-february-the-15th-build-day.md

```diff
@@ -37,14 +37,15 @@ After successfully removing an amount of burnt plastic from the extruder. We did
 
 The first image shows the “things and stuff” bought and the second one shows the print quality after the extruder has been un blocked (how it should look).
 
-[![20140220-191907.jpg](https://www.sheffieldhackspace.org.uk/wordpress/wp-content/uploads/2014/02/20140220-191907.jpg)](https://www.sheffieldhackspace.org.uk/wordpress/wp-content/uploads/2014/02/20140220-191907.jpg)
+![](/assets/blog/2014-02-20-february-the-15th-build-day/20140220-191907.jpg)
 
-[![20140220-191935.jpg](https://www.sheffieldhackspace.org.uk/wordpress/wp-content/uploads/2014/02/20140220-191935.jpg)](https://www.sheffieldhackspace.org.uk/wordpress/wp-content/uploads/2014/02/20140220-191935.jpg)
+![](/assets/blog/2014-02-20-february-the-15th-build-day/20140220-191935.jpg)
 
 **Arduino Robots**
 
 We also had a group member bring in an Arduino robot an AAR-04. It was programmed from the Arduino IDLE. There are two motors that drive the device back and forwards and a ball bearing at the front that keeps it in line. There are also some sensors at the front of the board, just by the ball bearing, these allow the robot to stop before it hit a surface.
 
 You can take a look at the video below of the robot working…
 
-<iframe allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="" frameborder="0" height="329" loading="lazy" src="https://www.youtube.com/embed/7jq4e9WCAcA?feature=oembed" title="Arduino AAR-04 at #shhm" width="584"></iframe>
\ No newline at end of file
+<iframe allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="" frameborder="0" height="329" loading="lazy" src="https://www.youtube.com/embed/7jq4e9WCAcA?feature=oembed" title="Arduino AAR-04 at #shhm" width="584"></iframe>
```

Comment:
Ill add a layout for youtube so they can be included without HTML

Reply:
Thanks Louise, that'll be a great help as quite a lot of the later posts have embedded youtube videos

Reply:
Thats in main now